config-templates/azure/azurefile-csi-driver/1.9.0/custom-parameters.json

```diff
@@ -0,0 +1,65 @@
+[
+    {
+        "description": "tenantId : The base64 encoded string generated from the azure.json config file.",
+        "displayname": "tenantId",
+        "name": "tenantId",
+        "required": "true",
+        "obfuscate": "true"
+    },
```

Comment:
```suggestion
```

Gaurav mentioned this is no longer needed/supported? - instead of passing this encoded string, they now pass params